### DISCUSSION TITLE
Moving a Columns to a new Project

### DISCUSSION THREAD

**Discussion by brookisme** [community/community#22739] [New to GitHub]
<p>Hi Everyone - I was wondering if there was a way to move a column from one project to new project.</p>
<p>I saw <a href="https://github.community/t/how-to-add-a-project-as-card-into-another-project-board/600/2">this</a> for copying cards but its not really so useful - as you still need to create a bunch of new cards.</p>
<p>The workflow I’m looking for is before I close a project to move all unfinished tasks to a single column, then when I open a new project, copy (or better yet move) the unfinished column to my new project.</p>
<p>If this isn’t possible - can we make it a feature request?</p>
<p>Thanks!</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/brookisme">@brookisme</a>,</p>
<p>Thanks for this feedback! We’re always working to improve GitHub and the GitHub Community Forum, and we consider every suggestion we receive. I’ve logged your feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.</p>
<p>Please let me know if you have any other questions.</p>
<p>Cheers!</p>

**Comment by chermaineslate** [comment]
<p>Any updates on this?</p>
